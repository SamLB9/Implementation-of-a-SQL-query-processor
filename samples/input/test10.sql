SELECT * FROM Car, Service WHERE Car.CarID = Service.CarID;