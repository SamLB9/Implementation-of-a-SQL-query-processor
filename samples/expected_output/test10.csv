1, 1, 2019, 1, 80000, 1, 1, 20200101, 500
1, 1, 2019, 1, 80000, 2, 1, 20200601, 300
2, 2, 2020, 1, 40000, 3, 2, 20200715, 400
3, 3, 2018, 2, 20000, 4, 3, 20200320, 350
4, 3, 2017, 2, 25000, 5, 4, 20200410, 450
5, 1, 2019, 3, 18000, 6, 5, 20200505, 200